src/app/checkout/navigateToOrderConfirmation.tsx

```diff
@@ -1,7 +1,14 @@
 import { noop } from 'lodash';
 
-export default function navigateToOrderConfirmation(location = window.location): Promise<never> {
-    const url = `${location.pathname.replace(/\/$/, '')}/order-confirmation`;
+export default function navigateToOrderConfirmation(location = window.location, orderId?: number): Promise<never> {
```

Comment:
By the time, `checkout-js` excutes ` navigateToOrderConfirmation()`, the order is finalised and the cart is gone. As a result, the `orderId` must be passed here.